source/parser/op.civet

```diff
@@ -140,7 +140,7 @@ function processBinaryOpExpression($0)
 
         // typeof shorthand: x instanceof "String" -> typeof x === "string"
         if op.token === "instanceof" and b.type === "Literal" and
-           b.children?.[0]?.type === "StringLiteral"
+           b.children is like [ {type: "StringLiteral"}, ... ]
```

Comment:
This one isn't that much clearer with 'is like'.

Maybe:

```
b is like {type: "Literal", children: [{type: "StringLiteral"}, ...]}
```

to include more of the structure of `b` in the check.